wiki/People/Tournament_Committee/en.md

```diff
@@ -26,13 +34,119 @@ Their responsibilities include:
+- Any staff and player lists are publicly available and the staff list is up-to-date.
+- The badge design does not contain any art owned by a third party.
+
+Assuming the tournament passes the review process, the assigned members will indicate that the tournament is eligible for a badge. The account support team will apply the badge at a later time to the winner(s) profile(s). If the tournament fails the review process, the reviewing members may advise the account support team on what changes need to be communicated to the tournament host(s). If these changes are not possible after the tournament’s conclusion, the Tournament Committee will hold an anonymous vote on whether to withdraw official support.
+
+If a tournament has been reported for violating the Official Support criteria and has submitted its conclusion email, and the Tournament Committee has deemed these reports actionable, the Tournament Committee will take the following course of action:
+
+- Determine if the issues brought up in the report break any of the Official Support criteria.
+- If they are severe enough to warrant such action, investigate the events detailed in the report to ensure the report’s validity and gather additional context.
+- Reach out to the infringing party, e.g. the tournament organiser, to get further information on the events detailed in the report.
+- After the investigation concludes and the Tournament Committee is considering administering relevant punishment, hold an anonymous vote to determine whether to do so and the severity of such punishment.
+
+### News Posts
+
+When an email requesting a news post for a tournament is received, the Tournament Committee will be notified. If the tournament satisfies all eligibility criteria for news posts and the consensus in discussion is positive after 48 hours, the Tournament Committee will forward the request to the news post team for approval.
+
+### Top Three Badge Support
+
+If a tournament host believes their tournament has achieved the highest level of production quality and organisation possible, they can request to receive badge prizes for all three finalists in their tournament. If a host requests to be awarded with this distinction, the assigned members will also notify the rest of the Tournament Committee for discussion. Following this discussion, an anonymous vote will be held.
+
+### Public Communication
+
+Public communication by the Tournament Committee is handled through several channels. The purposes for these sometimes overlap, but all are official sources of information kept up-to-date by the Tournament Committee.
+
+The Tournament Committee maintains an [account on X](https://x.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
```

Comment:
we don't go with the rename (you may notice this in news posts or citations in wiki articles). see https://github.com/ppy/osu-wiki/pull/9820

interestingly, x.com doesn't load for me but twitter.com does 🤔 

```suggestion
The Tournament Committee maintains an [account on Twitter](https://twitter.com/osu_tcomm) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
```

Reply:
god bless